bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/HvBattery.java

```diff
@@ -26,7 +27,7 @@
 public class HvBattery {
     public int hvBatteryLevel = UNDEFINED;
     public int distanceToHVBatteryEmpty = UNDEFINED;
-    public @NonNullByDefault({}) String hvBatteryChargeStatusDerived;
+    public @NonNullByDefault({}) StringType hvBatteryChargeStatusDerived;
```

Comment:
If this actually works then I have no problem with this change.

Reply:
Yes, it works, it's taken in charge by gson jsonDeserializer declaration.